typescript/src/languages/en-US/commands/management.json

```
@@ -357,21 +359,58 @@
+		"explainedUsage": [
+			[
+				"channel",
+				"A TextChannel. You can either put the name of the channel, tag it, or type in \"here\" to select the channel the message was sent."
+			]
+		],
+		"examples": [
+			"#member-logs",
+			"here"
+		],
+		"reminder": "All messages are in embeds so you will need to enable the permission **{{EMBED_LINKS, permissions}}** for Skyra."
+	},
+	"setMessageUpdateLogsDescription": "Set the message update logs channel.",
+	"setMessageUpdateLogsExtended": {
+		"usages": [
+			"here/TextChannel"
+		],
+		"extendedHelp": "This command helps you setting up the message update log channel, which will send messages each time a message is updated.",
+		"explainedUsage": [
+			[
+				"channel",
+				"A TextChannel. You can either put the name of the channel, tag it, or type in \"here\" to select the channel the message was sent."
+			]
+		],
+		"reminder": "All messages are in embeds so you will need to enable the permission **{{EMBED_LINKS, permissions}}** for Skyra.",
+		"examples": [
+			"#message-logs",
+			"here"
 		]
 	},
-	"setMessageLogsDescription": "Set the message logs channel.",
-	"setMessageLogsExtended": {
+	"setMessageDeleteLogsDescription": "Set the message delete logs channel.",
+	"setMessageDeleteLogsExtended": {
 		"usages": [
 			"here/TextChannel"
 		],
-		"extendedHelp": "This command helps you setting up the message log channel. A message log channel only sends three kinds of logs: \"Message Delete\", \"Message Edit\", and \"Message Prune\".\nAll messages are in embeds so you will need to enable the permission **{{EMBED_LINKS, permissions}}** for Skyra.\nYou also need to individually set the \"events\" you want to listen: `events.message-delete` and `events.message-edit` via the `config` command.",
+		"extendedHelp": "This command helps you setting up the message delete log channel, which will send messages each time a message is deleted.",
 		"explainedUsage": [
 			[
 				"channel",
 				"A TextChannel. You can either put the name of the channel, tag it, or type in \"here\" to select the channel the message was sent."
 			]
 		],
-		"reminder": "Due to Discord limitations, Skyra cannot know who deleted a message. The only way to know is by fetching audit logs, requiring the permission **{{VIEW_AUDIT_LOG, permissions}}** which access is limited in the majority of servers and the amount of times I can fetch them in a period of time.",
+		"reminder": "All messages are in embeds so you will need to enable the permission **{{EMBED_LINKS, permissions}}** for Skyra.\nDue to Discord limitations, Skyra cannot know who deleted a message. The only way to know is by fetching audit logs, requiring the permission **{{VIEW_AUDIT_LOG, permissions}}** which access is limited in the majority of servers and the amount of times I can fetch them in a period of time.",
```

I'm getting the impression that the code for fetching deleted user name is actually not there. If that's true then apply this suggestion:

```suggestion
		"reminder": "All messages are in embeds so you will need to enable the permission **{{EMBED_LINKS, permissions}}** for Skyra.\nDue to Discord limitations, Skyra cannot know who deleted a message.",
```


Otherwise apply this suggestion:
```suggestion
		"reminder": "All messages are in embeds so you will need to enable the permission **{{EMBED_LINKS, permissions}}** for Skyra.\nDue to Discord limitations, Skyra cannot know who deleted a message. If you do want deleted users to show be sure to give Skyra the **{{VIEW_AUDIT_LOG, permissions}}** permission.",
```

And what the heck is this?
> and the amount of times I can fetch them in a period of time.

---

She doesn't at the moment, but it's planned.

> and the amount of times I can fetch them in a period of time.

Ratelimit back-off.